spec/controllers/shipwire_webhook_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Spree::ShipwireWebhookController, type: :controller do
+  controller Spree::ShipwireWebhookController do
+    def create
```

Comment:
I think the point here is testing exactly an action not called `subscribe`.

Anyway I think `subscribe` specs are missing?